SoftLayer/CLI/virt/list.py

```diff
@@ -93,3 +93,23 @@ def cli(env, sortby, cpu, domain, datacenter, hostname, memory, network,
                        for value in columns.row(guest)])
 
     env.fout(table)
+
+    hardware_guests = vsi.get_hardware_guests()
+    for hardware in hardware_guests:
+        if 'virtualHost' in hardware and hardware['virtualHost']['guests']:
+            table_hardware_guest = formatting.Table(['id', 'hostname', 'CPU', 'Memory', 'Start Date', 'Status',
+                                                     'powerState'], title="Hardware(id = {hardwareId}) guests "
```

Comment:
to make this line a little easier to read, set the title to a variable and just use that here.

Reply:
Done.